config/config.yaml

```diff
@@ -117,6 +117,20 @@ datasets:
       # Relative path from the spras directory
       data_dir: "input"
 
+gold_standard:
+    -
+      label: gs0
+      node_files: ["gs_nodes0.txt"]
+      # edge_files: [] TODO: later iteration
+      data_dir: "input"
+      # Set of datasets (dataset labels) to compare with the specific gold standard dataset
+      datasets: ["data0"]
+    -
+      label: gs1
+      node_files: ["gs_nodes1.txt"]
+      data_dir: "input"
+      datasets: ["data1", "data0"]
```

Comment:
@agitter  what would you suggest the behavior should be if `datasets: []`.  Right now it can be empty and snakemake will not run evaluation.

Reply:
A warning or nothing is fine